lightspeed/modules/proc_create-lightspeed-instance.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: PROCEDURE
+
+[id="create-lightspeed-instance_{context}"]
+
+= Creating and deploying a {LightspeedShortName} instance
+
+Use this procedure to create and deploy a {LightspeedShortName} instance to your namespace.  
+
+.Prerequisites
+* You have created connection secrets for both {PlatformName} and {ibmwatsonxcodeassistant}.
+
+.Procedure
+
+. Go to {OCP}.
+. Select *Operators → Installed Operators*.
+. From the *Projects* list, select the namespace where you want to install the {LightspeedShortName} instance and where you created the connection secrets.
+. Locate and select the *{PlatformNameShort} (provided by Red Hat)* operator that you installed earlier.
+. Select *All instances → Create new*.
+. From the *Create new* list, select the *Ansible Lightspeed* modal.
```

Comment:
Should "modal" in this sentence be "module" or "model"?

Reply:
No, it's named as modal. Reference: https://access.redhat.com/documentation/vi-vn/red_hat_ansible_automation_platform/2.4/html-single/deploying_the_red_hat_ansible_automation_platform_operator_on_openshift_container_platform/index#assembly-install-aap-operator, so I'm not making any change here.